service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/JudgeAndLegalAdvisor.java

```diff
@@ -35,4 +35,12 @@ public static JudgeAndLegalAdvisor from(final Judge allocatedJudge) {
         }
         return judgeAndLegalAdvisorBuilder.build();
     }
+
+    @JsonIgnore
+    public JudgeAndLegalAdvisor resetJudgeProperties() {
```

Comment:
hmm, could we just have as `reset()`? As 1) its updating `JudgeAndLegalAdvisor` not `Judge` (different object) and 2) it's updating only two properties

Reply:
I'll make this change too in my PR